luciana.txt

```diff
@@ -0,0 +1,3 @@
+Luciana Ribeiro
+Engenheira de Software
+24 anos
```

Comment:
Coloca mais informações

Reply:
Just added new informations, can you take a second look? @KiSobral

Reply:
Estou refazendo a revisão agora